sources/game/Game.gd

```diff
@@ -111,3 +125,21 @@ func puzzle_quit() -> void:
 	puzzle.queue_free()
 
 	puzzle = null
+
+
+"""Handle pause menu events
+"""
```

Comment:
I believe level.get_tree() is the same as get_tree() so you can avoid unnecessary checks here. They are part of the same SceneTree